res/text/bms_page_es-419.txt

```diff
@@ -0,0 +1,9 @@
+Modo
+Normal
+Demostración
+Atrás
+Análisis de beatmaps
+Error al leer Be-Music Source
+Debe haber al menos un beatmap osu!/nmania válido en la carpeta. Además, el nombre de la carpeta no debe ser demasiado largo ni contener caracteres especiales.
+Error al leer el fondo de Beatmap
+Error al leer Be-Music Source
```

Comment:
With #18 , there will be an extra (the 10th) line "Difficulty adjustment" (title for choice, do player want to decrease/increase game's difficulty). Please, add it to make this file usable with next release.